cogs/quote_commands.py

```diff
@@ -36,7 +36,10 @@ async def quote(self, ctx:utils.Context, messages:commands.Greedy[discord.Messag
 
         # Make sure a message was passed
         if not messages:
-            return await ctx.send("I couldn't find any references to messages in your command call.")
+            if ctx.message.reference is not None:
+                messages = [ctx.message.reference.message_id]
```

Comment:
`messages` needs to be a list of message objects rather than message IDs - you should fetch the message from the API to put into this list.

Reply:
ah

Reply:
done 337a35fe266433ba7aa0961c4743d78bc0e6af6d